Jakub_Murszewski_not_MA_thesis_overview.md

```diff
@@ -0,0 +1,237 @@
+
+For the programme to work correctly, several additional files need to be present. Those are:
+
+* *.txt* file with negative sentiment words
+* *.txt* file with positive sentiment words
+* *.txt* file with stopwords
+
+All of which need to be located in the same folder as the main programme file.
+
+**The code and necessary .txt files can be downloaded from:**
+
+[Here](https://github.com/MurszewskiJ/SSAT)
+
+# User Guide
+
+
+**First thing to appear after opening the program is the following menu:**
+
+![Menu][Menu]
+
+To access one of the functions of SSAT enter the number associated wit the function and press *ENTER*
+
+### Functions
+
+
+**"1. Sentiment Analysis (with stopwords)"**
+
+
+This function will analyse the whole selected text without deleting any of its contents.
+
+Upon selection of this option, the following will appear:
+
+![image_1][image_1]
+
+To proceed, enter:
+* either the path to the file which you want to analyse (if the file is not in the same folder as the programme
+* or only the name of the file (if the file is in the same folder as the programme)
+
+
+**Note that the file you wish to analyse should be in .txt format.**
+
+Then, press *ENTER*.
+
+
+
+The program will analyse the text and display the result (i.e. the overall sentiment of the text).
+
+To see the details and charts, type "Y" in the following prompt:
+
+![image_2][image_2]
+
+Then, press *ENTER*.
+
+Then, the programme will display the following:
+
+* Percentage value of positive words
+* Percentage value of negative words
+* Percentage value of neutral words
+* List of most common negative words
+* List of most common positive words
+* List of most common neutral words
+* Chart containing the percentage value of the overall sentiment
+* Chart with the most common positive words (up to 10)
+* Chart with the most common negative words (up to 10)
+* Chart with the most common neutral words (up to 10)
+
+To save this data to a file, type "Y" in the following prompt:
+
+![image_3][image_3]
+
+Then, press *ENTER*.
+
+Then, enter the name of the file in the following prompt:
+
+![image_4][image_4]
+
+**The name of the file should end with _".txt"_.**
+
+Then, press *ENTER.*
+
+The programme will then proceed to save the data to the following file, which will be then localised in the same folder as the programme.
+
+**"2. Sentiment Analysis (without stopwords)"**
+
+This function will analyse the selected text after deleting stopwords from it.
+
+This function shares its operations with the previous function. For a step-by-step guide, see **"1. Sentiment Analysis (with stopwords)"**.
+
+
+
+**"3.Keywords Analysis (with stopwords)"**
+
+This function will search the text for sentences with the chosen keyword.
+
+Upon selecting this option, the following will appear:
+
+![image_5][image_5]
+
+To proceed, enter tke keyword you wish to search for.
+
+**Remember that it is case sensitive.**

```

Comment:
Ok, it's good that you're warning the user, but should it be like this? I'm not sure if case sensitivity makes sense in sentiment analysis - 'hate' and 'HATE' are equally negative, I think.